README.markdown

```diff
@@ -50,6 +50,16 @@ This library is production ready.
 Synopsis
 ========
 
+This library is automatically loaded by default in OpenResty 1.15.8.1. This
```

Comment:
Will be updated for the 1.17.1.1 release mentioning it is mandatory (if we remove the CFunction API by then).